modules/ui/sidebar.js

```diff
@@ -54,6 +54,12 @@ export function uiSidebar(context) {
             .on(_pointerPrefix + 'down.sidebar-resizer', pointerdown);
 
         var downPointerId, lastClientX, containerLocGetter;
+        var flagCollapse = false;
+        var flagExpand = false;
+
+        var holdWindowSize = 0;
+
+        d3_select(window).on('mouseover', function(){checkWindowWidth()}); //event listener
```

Comment:
Why not trigger this in the [resize](https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event) event?